templates/web/common/site_layout.tt.html

```diff
@@ -490,6 +490,14 @@ <h3 class="title-5 text-medium">[% lang('footer_discover_the_project') %]</h3>
 [% END %]
 
 </body>
+<script src="[% static_subdomain %]/js/dist/JsBarcode.all.min.js"></script>
+<script>
+	//init barcode SVGs if exist on the page
+	let barcode_svg_elements = document.getElementsByClassName("barcode-svg"); 
+	if(barcode_svg_elements.length!=0){
+		JsBarcode(".barcode-svg").init();	
```

Comment:
Since you already have the array, this should be enough:
```suggestion
		JsBarcode(barcode_svg_elements).init();	
```
`JsBarcode` seems to do the same by calling [`getRenderProperties`](https://github.com/lindell/JsBarcode/blob/5ed2a2b9da5f82da3c6159eb47a21d06f1cf797d/src/help/getRenderProperties.js#L21) internally. Might save some ms by not running the selector again